docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,174 @@
+# Getting Started
+
+In this section, we will show you how to install and configure Gateway API and Istio via the Ingress Operator. We will
+configure a simple HTTPRoute and backend to demo basic Gateway API functionality.
+
+## Prerequisites
+
+* A non-production OpenShift Cluster, version v4.13.z or greater, on a cloud platform that supports Kubernetes external
+  load balancers
+* If OSSM is already installed, it must be OSSM 2.4.0
+* If OSSM is already installed, you cannot have any existing SMCPs
+
+## Installation via the Ingress Operator
+To install Gateway API using the Ingress Operator, you will do the following:
+1. Grant the Ingress Operator the cluster-admin role
+2. Enable the feature gate
+3. Create a GatewayClass with the controller name “openshift.io/gateway-controller”
+
+Let’s walk through each of these steps in more detail.
+
+First, grant the Ingress Operator the cluster-admin role. Currently, OSSM requires the cluster-admin role to create SMCPs:
+```shell
+$ oc adm policy add-cluster-role-to-user cluster-admin -z ingress-operator -n openshift-ingress-operator
+```
+
+Enable the feature gate for Gateway API. This will instruct the cluster Ingress Operator to install the GatewayClass CRD:
+```shell
+$ oc patch featuregates/cluster --type=merge --patch='{"spec":{"featureSet":"CustomNoUpgrade","customNoUpgrade":{"enabled":["GatewayAPI"]}}}'
+```
+
+Next, create the GatewayClass with the controller name “openshift.io/gateway-controller”. This will instruct the Ingress Operator to create a SMCP and install the Gateway, HTTPRoute, and Reference Grant CRDs:
+```shell
+$ oc create -f -<<'EOF'
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: GatewayClass
+metadata:
+  name: openshift-default
+spec:
+  controllerName: openshift.io/gateway-controller
+EOF
+```
+
+Confirm the Ingress Operator has installed the HttpRoute, Gateway, and ReferenceGrant CRDs:
+```shell
+$ oc get crds httproutes.gateway.networking.k8s.io gateways.gateway.networking.k8s.io referencegrants.gateway.networking.k8s.io
+NAME                                        CREATED AT
+httproutes.gateway.networking.k8s.io        2023-04-04T18:02:55Z
+gateways.gateway.networking.k8s.io          2023-04-04T18:02:55Z
+referencegrants.gateway.networking.k8s.io   2023-04-04T18:02:56Z
+```
+
+Confirm the Ingress Operator has created the SMCP and it is ready:
```

Comment:
This took 40 seconds on OpenShift 4.14.  May want to tell them to wait for it.

Reply:
Done.

Reply:
Yea I actually ran this myself recently, it takes a long time to show up, I'll be more specific.

Reply:
Done